src/utils/testConstants.jsx

```diff
@@ -5,7 +5,7 @@ import thunkMiddleware from 'redux-thunk';
 import accessibility from '../data/reducers/accessibility';
 import { assets, metadata } from '../data/reducers/assets';
 
-const courseDetails = {
+export const courseDetails = {
```

Comment:
I didn't think it made sense for this to be a default export instead of just a named export like the rest of the test constants.